spec/features/api/pages_spec.rb

```diff
@@ -229,7 +229,30 @@
     vetted_stasuses.include?(Vetted.trusted.id).should == true
     vetted_stasuses.include?(Vetted.untrusted.id).should == false
   end
-
+  it 'pages should filter out trusted and untrusted objects' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.